### ISSUE TITLE
[Feature] Add Vite-Builder Support

### ISSUE THREAD

**Issue by Gaubee** [storybookjs/addon-svelte-csf#64]
Vite-Builder is one of the office builder, pls add support.❤️

**Comment by benmccann** [comment]
I think there's already support. You can see an example project here:

https://github.com/michaelwooley/storybook-experimental-vite

**Comment by konstantinblaesi** [comment]
@benmccann when doing 
```
npm install
npm run storybook
```
for that repository, I see traces of webpack. Is storybook somehow ditching the vite builder once this package is installed/enabled/used?

**Comment by benmccann** [comment]
It uses Vite, but is still pulling in webpack anyway. Should hopefully be fixed in Storybook 7

**Comment by IanVS** [comment]
That's right, in Storybook 6, webpack is still used to build the "manager" part of the UI, which no longer happens in SB 7.

**Comment by IanVS** [comment]
@j3rem1e what would you think about adding vite-builder support into this addon?  Currently, we have a [custom vite-plugin](https://github.com/storybookjs/storybook/blob/next/code/frameworks/svelte-vite/src/plugins/csf-plugin.ts) in the `svelte-vite` framework (which I realized recently isn't being used, though it is in 6.5), which imports some files from this addon and duplicates some of what this addon is doing.  

What would you think about making this addon-svelte-csf package a bit more general, including the vite plugin here, and adding a `viteFinal` alongside the current webpack config?  It would mean a few more optional peer dependencies perhaps, but it might be nice to have the related logic all in one place, and one addon to tell people to install.  What do you think?  Should I work on a PR?

**Comment by j3rem1e** [comment]
@IanVS yep, that's a good idea ^^

**Comment by benmccann** [comment]
https://github.com/storybookjs/addon-svelte-csf/pull/72 was merged and said it closes this issue. I'm confused as to why GitHub didn't auto-close this issue. Maybe it was just a GitHub glitch? Should this issue be closed or is there more to do here?

**Comment by IanVS** [comment]
I think the problem is that I used the storybook monorepo convention of `issue: #xyz` instead of `closes: #xyz`, and the same scripts that run there, don't run here.  I do believe this can be closed now, thanks for mentioning it.
